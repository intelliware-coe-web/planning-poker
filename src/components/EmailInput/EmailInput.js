import React from 'react';

function EmailInput(props) {
    return (
        <div className="uk-margin">
            <div className="uk-inline">
                <span className="uk-form-icon" uk-icon="icon: mail"></span>
                <input className="uk-input" type="email" placeholder="Email" />
            </div>
        </div>
    );
}

export default EmailInput;