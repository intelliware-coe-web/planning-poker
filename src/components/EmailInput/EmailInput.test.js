import React from 'react';
import EmailInput from './EmailInput';

it('email input should match snapshot', () => {
    expect(EmailInput()).toMatchSnapshot();
});