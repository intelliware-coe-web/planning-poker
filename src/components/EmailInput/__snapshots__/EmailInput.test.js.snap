// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`email input should match snapshot 1`] = `
<div
  className="uk-margin"
>
  <div
    className="uk-inline"
  >
    <span
      className="uk-form-icon"
      uk-icon="icon: mail"
    />
    <input
      className="uk-input"
      placeholder="Email"
      type="email"
    />
  </div>
</div>
`;
